src/test/app/MPToken_test.cpp

```diff
@@ -1648,7 +1649,6 @@ class MPToken_test : public beast::unit_test::suite
     testTxJsonMetaFields(FeatureBitset features)
     {
         // checks synthetically parsed mptissuanceid from  `tx` response
```

Comment:
I think I know what my confusion was earlier. It's the word "parsed". 
```suggestion
        // checks synthetically injected mptissuanceid from `tx` response
```